package.json

```diff
@@ -96,7 +96,7 @@
 					},
 					"camelk.yaml.schema": {
 						"type": "string",
-						"default": "https://raw.githubusercontent.com/apache/camel-k-runtime/master/camel-k-loader-yaml/camel-k-loader-yaml/src/generated/resources/camel-yaml-dsl.json",
+						"default": "https://raw.githubusercontent.com/apache/camel-k-runtime/camel-k-runtime-parent-1.5.0/camel-k-loader-yaml/camel-k-loader-yaml/src/generated/resources/camel-yaml-dsl.json",
```

Comment:
Now we won't detect problems in newer versions right away. Is that wanted behavior?

Reply:
it will avoid to break on already released version.

There won't be undetected breakage as it will be detected when the url is updated.

We are not testing the content of the json schema. This extension is not responsible for it. We are testing that we can retrieve the json schema.